projects/vdk-plugins/vdk-meta-jobs/README.md

```diff
@@ -123,9 +123,9 @@ def run(job_input: IJobInput) - > None:
 
 The depends_on key stores the dependencies of each job - the jobs that have to finish before it starts.
 The DAG execution starts from the jobs with empty dependency lists - they start together in parallel.
-But what happens if they are too many? It could cause server overload. In order to avoid such unfortunate situations,
-a limit in the number of concurrent running jobs is set. This limit is
-a [configuration variable](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_configuration.py#L87)
+<br>But what happens if they are too many? It could cause server overload.
```

Comment:
nit: why having HTML in favour of MD markup? We may want to stick to a human-readable markup in cases possible, preventing complications.

Reply:
Good one, thanks. Idk why I went this way. Fixed now.